### PR TITLE
add a blog post: Using WildFly Glow to provision a WildFly server for a RESTEasy based project 

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
@jamezp Could you please help to review this blog post when you have time? Thanks!

File: _posts/2024-01-31-resteasy-glow.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+I’ll give some brief description to the above configuration. To use the Glow feature, first we need to add the following configuration to the `wildfly-maven-plugin`:
+
+```xml
+<discover-provisioning-info>
+    <version>${version.wildfly}</version>
+    <failsOnError>false</failsOnError>
+</discover-provisioning-info>
+```
+
+The above `discover-provisioning-info` will enable the embedded Glow feature to scan the project source code and detect the layers to be used in the provisioned WildFly server. Next in the `executions` section of the plugin, the `package` goal needs to be added:
```
Comment (jamezp) [inline _posts/2024-01-31-resteasy-glow.md:22]: We might want to add a link to https://www.wildfly.org/news/2024/01/29/wildfly-glow/ for Glow.

File: _posts/2024-01-31-resteasy-glow.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    </executions>
+</plugin>
+<plugin>
+    <artifactId>maven-surefire-plugin</artifactId>
+    <configuration>
+        <excludes>
+            <exclude>**/*IT.java</exclude>
+        </excludes>
+    </configuration>
+</plugin>
```
Comment (jamezp) [inline _posts/2024-01-31-resteasy-glow.md:83]: This actually shouldn't be required. The default includes does not include `*IT.java`.

File: _posts/2024-01-31-resteasy-glow.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+[INFO] Some suggestions have been found. You could enable suggestions with the --suggest option (if using the WildFly Glow CLI) or <suggest>true</suggest> (if using the WildFly Maven Plugin).
+[INFO] Provisioning server in /Users/weli/works/resteasy-wildfly/target/server
+[INFO] Resolving feature-packs
+[INFO] Installing packages
+[INFO] Resolving artifacts
+[INFO] Generating configurations
+...
+```
+
+From the above output we can see that the `jaxrs` layer is detected automatically by the Glow scanning feature. The layer is detected by scanning the source code of the sample project, and it found the JAX-RS annotations in the source code, so the `jaxrs` layer is added in the provisioned server. Then the provisioned server will be started, and the integration tests will be run in the server, and finally the server will be stopped after the integration testing phase.
```
Comment (jamezp) [inline _posts/2024-01-31-resteasy-glow.md:116]: "JAX-RS annotations" should probably be "Jakarta REST annotations"

**Comment by liweinan** [comment]
Thanks for the review James! I'll modify the blog post accordingly.

**Comment by liweinan** [comment]
@jamezp I have updated this blog post and the relative sample project according to your review comments. Could you please help to check if anything else needs to be modified? Thanks!

**Review by jamezp — APPROVED** [review]

**Pull request merged**